src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html

```diff
@@ -62,55 +77,377 @@ <h3 class="u-text-body-bold">Badge Basics</h3>
+							justify-content: center;
+							align-items: center;
+							display: flex;
+						"
+					>
+						<span
+							style="
+								color: black;
+								font-weight: 600;
+								padding-left: 1em;
+								padding-right: 1em;
+								padding-top: 0.5em;
+								padding-bottom: 0.5em;
+								font-size: 28px;
+								line-height: 55.2px;
+							"
+							>KI-BADGE-ASSISTENT NUTZEN</span
+						>
+					</div>
+					<div
+						style="
+							display: flex;
+							justify-content: space-between;
+							width: 100%;
+							padding: 4em;
+							margin-left: 2em;
+						"
+					>
+						<div style="display: flex; flex-direction: column">
+							<h2 style="color: #ffffff">Meine inhaltliche Beschreibung</h2>
+							<textarea style="background-color: white; border-radius: 10px"> </textarea>
+							<div style="color: white">
+								<label for="kursdauer"> Dauer des Kurses</label>
+								<div style="display: flex; gap: 2em">
+									<input
+										style="width: 4em; background-color: white"
+										type="number"
+										id="kursdauer"
+										name="kursdauer"
+										value="170"
+									/>
+									<span>Minuten</span>
+								</div>
+								<button
+									class="button"
+									style="
+										background-color: var(--color-grün);
+										color: var(--color-blue);
+										margin-top: 2em;
+									"
+								>
+									<span style="font-weight: 400">Passende</span>
+									<span style="font-weight: 700"> Kompetenzen vorschlagen </span>
+								</button>
+							</div>
+						</div>
+						<div style="width: 50%">
+							<h2 style="color: white">Passende Kompetenzen</h2>
+							<div style="background-color: white; border-radius: 10px; width: 75%; padding: 1em">
+								<p style="width: 100%; font-style: italic">
+									Hier erscheinen passende Kompetenzen, sobald du die Beschreibung deines Lehrinhalts
+									in das Textfeld links eingefügt hast und auf "Pasende Kompetenzen vorschlagen"
+									klickst.
+								</p>
+							</div>
+						</div>
+					</div>
+				</div>
+			</li>
+			<hr style="border: 1px dashed var(--color-blue); margin-top: 2em; margin-bottom: 2em" />
+			<li>
+				<span style="font-size: 28px; line-height: 55.2px; color: var(--color-blue)"
+					>Händisch Kompetenzen hinzufügen</span
+				>
+				<p>
+					Alternativ oder ergänzend kannst du händisch Kompetenzen hinzufügen. Gerne kannst du als
+					Unterstützung die Suchfunktion von ESCO nutzen, um die passende Kompetenz zu finden und die
+					Informationen per copy&paste in die unten stehenden Felder übertragen.
+				</p>
+				<button style="margin-top: 2em; background-color: #492e98" class="button">
+					<a
+						style="color: white"
+						href="https://esco.ec.europa.eu/en/classification/skill_main"
+						target="_blank"
+						>Bring mich zu ESCO</a
+					>
+				</button>
+
+				<!-- <ng-container *ngIf="badgeClassForm.controls.competencies.controls.length === 0; else competencyLoop">
```

Comment:
What is this commented code doing here?

Reply:
There is a comment above it now, explaining it

Reply:
Hm but does this TODO have its own ticket then?

Reply:
Now it [does](https://github.com/orgs/mint-o-badges/projects/2/views/1?pane=issue&itemId=57524205)